index.html

```diff
@@ -64,5 +64,183 @@ <h2 class="container job-title">I am a Software developer </h2>
             </ul>
         </div>  
     </section>
+    <!-- Recent work section -->
+
+<section class="recent-works">
+    <div class="first">
+        <h3>My Recent Works</h3>
+        <hr>
+    </div>
+    <div class="recent-work" id="recent-works">
+        <a href="" class="women"><img src="assets/Img_Placeholder.png" alt="women doing yoga"></a></div>
+    <div>
+        <h4>Multi-Post-Stories</h4>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+        <ul class="tech-used">
+            <li class="css">Css</li>
+            <li class="HTML">html</li>
+            <li class="boots">Bootstrap</li>
+            <li class="ruby">Ruby</li>  
+        </ul>
+        <form>
```

Comment:
- [Optional] Please I would suggest the `<form>` tag is not needed here.

Reply:
@ZuhRa-HashiMi please add a proper description for this PR and fix your Linter error before you make another PR